docs/releases/openmathinstruct2/training.md

```
@@ -49,73 +49,66 @@ python -m nemo_skills.training.prepare_data \
+    ++checkpointing.save_period=3330 \
+    ++sft.max_num_steps=20000 \
+    ++sft.max_num_epochs=100 \
+    ++policy.sequence_packing.enabled=False \
+    ++checkpointing.checkpoint_must_save_by=00:03:20:00
 ```
 
+
 If you have a job timeout, it's necessary to set the maximum time per run to 40 minutes
 before the timeout to allow for the final checkpoint to be saved. E.g. if your timeout is 4 hours,
-add `++exp_manager.max_time_per_run=00:03:20:00`
+add `++checkpointing.checkpoint_must_save_by=00:03:20:00 `
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)